app/concepts/todo_list/create.rb

```diff
@@ -0,0 +1,32 @@
+class TodoList::Create < Trailblazer::Operation
+  include Model
+  model TodoList, :create
+
+  contract do
+    property :name
+    property :user_id
+  end
+
+  def process(params)
+    validate(params[:todo_list]) do |f|
+      id, uid = params[:todo_list][:id], params[:todo_list][:user_id]
```

Comment:
Is this deconstruction of `params` a responsibility of the `process` method or is it better placed inside the `find_or_create_user` method?